DESCRIPTION

```diff
@@ -20,11 +19,15 @@ URL: https://github.com/jolars/sgdnet
 BugReports: https://github.com/jolars/sgdnet/issues
 RoxygenNote: 6.0.1
 Roxygen: list(markdown = TRUE)
+Depends: R (>= 3.5.0)
```

Comment:
Is this a hard dependency or just we're not testing for anything earlier?

Reply:
There's no real reason other than that I'm developing on 3.5.0, which makes it convenient. It looks like we can drop to 3.3.0 without any issues. I have no strong feelings about this.

Reply:
would be good to drop this dep if possible, to allow people with older R to use our pkg

Reply:
Done